libpod/options.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/containers/storage/pkg/idtools"
 	"github.com/cri-o/ocicni/pkg/ocicni"
 	"github.com/pkg/errors"
+	"net"
```

Comment:
Add builtin golang to the top not to the github section.

Reply:
fixed